packages/components/src/dropdown-menu/index.js

```diff
@@ -153,7 +153,8 @@ function DropdownMenu( {
 										},
 									) }
 									icon={ control.icon }
-									role="menuitem"
+									aria-checked={ ( control.role === 'menuitemcheckbox' || control.role === 'menuitemradio' ) ? control.isActive : undefined }
```

Comment:
Is there no active state for other roles?

Reply:
No, for the normal menuitem, there is no aria-checked or the like, that is only supported for menuitemcheckbox and menuitemradio. And those are the only roles allowed here.